overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -10189,14 +10190,14 @@
           "lockedprogress:1": 1,
           "name:8": "Nitrogen Dioxide",
           "partysinglereward:1": 0,
-          "questlogic:8": "AND",
+          "questlogic:8": "OR",
           "repeat_relative:1": 1,
           "repeattime:3": -1,
           "simultaneous:1": 0,
           "snd_complete:8": "minecraft:entity.player.levelup",
           "snd_update:8": "minecraft:entity.player.levelup",
           "tasklogic:8": "AND",
-          "visibility:8": "ALWAYS"
```

Comment:
Why did you hide Nitrogen Dioxide?

Reply:
> Why did you hide Nitrogen Dioxide?

The thought was that it's not required for anything in the tab, it's the most energy efficient option for Nitric Acid but the LCR recipe is also valid. I'll unhide it and just keep the dependency link to Nitric Acid as mouseover only, to reduce unnececssary lines.

Reply:
Yep, makes sense, but since the quest still has things depending on it and it has a task, it won't work properly. Keep it unhidden for now, we can work out if we should remove it in the future.